README.md

```diff
@@ -202,6 +212,8 @@ IEEE Trans. Neural Netw. Learn. Syst., 28(4):791-803, 2017.
 - <a name="Ramesh17arxiv"></a>Ramesh, B., Yang, H., Orchard, G., Le Thi, N.A., Xiang, C,  
 *[DART: Distribution Aware Retinal Transform for Event-based Cameras](https://arxiv.org/pdf/1710.10800.pdf)*,  
 arXiv:1710.10800, 2017.
+- <a name="Belbachir11tie"></a>Belbachir, A., Hofstaetter, M., Litzenberger, M., Schoen, P., *[High Speed Embedded Object Analysis Using a Dual-Line Timed-Address-Event Temporal Contrast Vision Sensor](https://doi.org/10.1109/TIE.2010.2095390)*,
```

Comment:
reformat entry (3 separate lines, as the rest of the entries) and sort chronologically (an entry from 2011 should not be after an entry from 2017).

Reply:
OK